### ISSUE TITLE
Implement Undo functionality upon a zone/playlist removal

### ISSUE THREAD

**Issue by elias-pap** [UoC-Radio/flow-dashboard#4]
An Undo nautilus-like pop-up message should be displayed every time a playlist or a zone is deleted, giving the user the opportunity to undo the action.

**Comment by ggalan87** [comment]
Is there a confirmation pop-up window when delete is pressed? I don't remember

**Comment by elias-pap** [comment]
Yes. It looks like [this](https://forum.manjaro.org/uploads/default/original/3X/b/c/bc0af05db1be4be49fe08aa4627511748b9f207d.png).
